## 05_hppabuild by Junichi Uekawa <dancer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: patch to get HPPA Linux build. applied upstream on 3.7p1

--- canna-3.7.orig/Canna.conf.dist
+++ canna-3.7/Canna.conf.dist
@@ -183,7 +183,7 @@
 #if defined(cygwinArchitecture)
 /* import libraryの方を指定しておく。DLL本体は/usr/X11R6/binに入る。 */
 sharedLibExtension = a
-#elif defined(HPArchitecture)
+#elif defined(HPArchitecture) && !defined(__linux__)
 sharedLibExtension = sl.$(cannaDsoRev)
 #elif defined(DarwinArchitecture)
 sharedLibExtension = $(cannaDsoRev).dylib
